static/main.js

```diff
@@ -67,13 +60,21 @@ function init() {
         try {
             katex.render(input.value, math, options);
         } catch (e) {
-            if (e.__proto__ == katex.ParseError.prototype) {
+            if (e.__proto__ === katex.ParseError.prototype) {
                 console.error(e);
             } else {
                 throw e;
             }
         }
     }
+
+    if (module.hot) {
+        module.hot.accept('../katex.webpack.js', reprocess);
+    }
+}
+
+if (/^\/(?:index.html)?$/.test(location.pathname)) {
```

Comment:
I wonder there is a better way to load this code and enable HMR only in the main page(`static/index.html`).